### PR TITLE
python v3.10.15

### PR DESCRIPTION
**Pull request by h-vetinari**

### CODE REVIEW

**Comment by isuruf** [comment]
Thanks

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by h-vetinari** [comment]
Waiting for https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/342 to be fixed

File: recipe/yum_requirements.txt [hunk 1]
```diff
@@ -1,5 +1,2 @@
+#tk_needs_libx11_but_avoids_a_runtime_requirement_of_libx11_to_reduce_the_dependencies_of_python
 libx11
```
Comment (h-vetinari) [inline recipe/yum_requirements.txt:2]: You need at least
```suggestion
libX11-devel
```
however, this kind of setup failed previously (see [logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1028624&view=logs&jobId=2aeac593-170e-54b6-7054-3a1104dd2f98) for https://github.com/conda-forge/python-feedstock/pull/706/commits/165b52f39c47aaa7a436a86e026ac841ccf2668c)

**Comment by h-vetinari** [comment]
`libxcb` is not a heavy dependency, it's about 400kb including transitive deps.

**Comment by isuruf** [comment]
`libxcb` is not a dependency of python nor tk. `libxcb` is a dependency of `libx11` which is a dependency of `tk`.

**Pull request merged**